third_party/lasread/lidario.go

```diff
@@ -479,8 +480,23 @@ func (las *LasFile) readHeader() error {
 	offset += 8
 	las.Header.MinZ = math.Float64frombits(binary.LittleEndian.Uint64(b[offset : offset+8]))
 	offset += 8
-	if las.Header.VersionMajor == 1 && las.Header.VersionMinor == 3 {
+	if las.Header.VersionMajor == 1 && las.Header.VersionMinor >= 3 {
```

Comment:
The added headers are LAS 1.4 specific, they are not present in LAS 1.3 specs so we need an additional condition check here.

Reply:
According to the screenshot provided above, there is indeed a small problem with this piece, I will adjust it later. Besides, I don't have the LAS1.3 file at present, so I need to find the LAS1.3 file for testing.Please wait :)